doc/articles/getting-started-counter.md

```diff
@@ -0,0 +1,322 @@
+
+In this tutorial you will learn how to:
+
+- Create a new Project with Uno Platform using Visual Studio Template Wizard or the `dotnet new` command
+- Add elements to the XAML file to define the layout of the application
+- Add code to the C# file to implement the application logic using the Model-View-ViewModel (MVVM) pattern
+- Use data binding to connect the UI to the application logic
+
+To complete this tutorial you don't need any prior knowledge of the Uno Platform, XAML or C#. However, if you're a more experienced developer, you may want to skip this tutorial and jump straight to either the [SimpleCalculator](xref:Workshop.SimpleCalc.Overview) or the [TubePlayer](xref:Workshop.TubePlayer.Overview) sample. Both are more complex and will give you a better idea of what Uno Platform can do.
+
+## Create the Application
+
+To get started, we're going to use the Uno Platform solution template with the simplest set of options selected. The solution template can be accessed using either Visual Studio's New Project wizard or the command line.
+
+# [Visual Studio](#tab/vs)
+
+> [!NOTE] 
+> If you don't have the `Uno Platform Extension for Visual Studio` installed, follow [these instructions](xref:Uno.GetStarted.vs2022).
+
+- Launch `Visual Studio` and click on `Create new project` on the Start Window. Alternatively, if you're already in Visual Studio, click `New, Project` from the `File` menu.
+
+- Type `Uno Platform` in the search box
+
+- Click `Uno Platform App`, then `Next`
+
+- Name the project `Counter` and click `Create`
+
+At this point you'll enter the `Uno Platform Template Wizard`, giving you options to customize the generated application. For this tutorial we're only going to configure the markup language and the presentation framework.
+
+- Select `Basic` and click `Customize`
+
+- Select the `Presentation` tab and choose `MVVM`
+
+- Select the `Markup` tab and choose `XAML`
+
+Before completing the wizard, take a look through each of the sections and see what other options are available. You can always come back and create a new project with different options later. For more information on all the template options, see [Using the Uno Platform Template](xref:Uno.GettingStarted.UsingWizard).
+
+- Click `Create` to complete the wizard
+
+The template will create a new solution with a number of projects. The main project is a class library called `Counter` which contains the application code. The other projects are platform-specific heads that contain the platform-specific code required to run the application on each platform.
+
+
+# [Command Line](#tab/cli)
+
+> [!NOTE] 
+> If you don't have the Uno Platform dotnet new templates installed, follow [these instructions](xref:Uno.GetStarted.dotnet-new).
+
+From the command line, run the following command:
+
+```
+dotnet new unoapp -preset blank -presentation mvvm -markup csharp -o Counter
```

Comment:
above you mentioned using the `XAML` as `Markup` and here it's the `C#Markup`, that right?

 ```suggestion
dotnet new unoapp -preset blank -presentation mvvm -o Counter
```